src/utils/createAsset.ts

```diff
@@ -118,40 +16,40 @@ export const createAsset = (
   const svgPath = (() => {
     if (_asset.parts) {
       return _asset.parts
-        .map((part: any) => {
-          const path = normalizePath(part.body, width);
+        .map((part: OriginalAssetPart) => {
+          const path = normalizePath(part.body as string, width);
```

Comment:
@snakajima 
Because part.body is string | Uint8Array, cast is needed.
In the case of Uint8Array, it may be better to put it in part.data etc.